src/core/public/plugins/plugins_service.test.ts

```diff
@@ -274,7 +274,8 @@ describe('PluginsService', () => {
         jest.useRealTimers();
       });
 
-      it('throws timeout error if "setup" was not completed in 30 sec.', async () => {
+      // Flaky test
```

Comment:
I believe the flaky tests is kind of a misnomer. These are always failing for me.

I can see upgrading to jest v27 there was a couple of regressions (really default behavior change).
https://github.com/nock/nock/issues/2200
https://github.com/facebook/jest/issues/11539

I was able to get this test passing by updating line 271 in this file from:
`jest.useFakeTimers();` ~> `jest.useFakeTimers('legacy');`

and adding:
`import { setImmediate } from 'timers';`

to this file.

Reply:
Oh nice. At some point I attempted changing the parameter for `useFakeTimers` before, but I didn't have the import. Thanks!